superset-frontend/src/dashboard/components/menu/MarkdownModeDropdown.tsx

```diff
@@ -46,6 +46,7 @@ export default class MarkdownModeDropdown extends PureComponent<MarkdownModeDrop
 
     return (
       <PopoverDropdown
+        data-testid="markdown-mode-dropdown"
```

Comment:
Superset uses [data-test](https://github.com/apache/superset/blob/master/superset-frontend/spec/helpers/setup.ts#L26) for the testid attribute
```suggestion
        data-test="markdown-mode-dropdown"
```